changelog/@unreleased/pr-4973.v2.yml

```diff
@@ -0,0 +1,8 @@
+type: improvement
+improvement:
+  description: |-
+    Targeted sweep runtime configuration now has a `enableAutoTuning` parameter. While true, targeted sweep will dynamically adjust the pause between sweep iterations based on the amount of work that was done. It will, furthermore, attempt to read a full batch (100k) of entries to sweep whenever possible.
```

Comment:
Probably worth calling out this is not yet enabled by default and please talk to the AtlasDB team if you want to use it.